<launch>
    <node name="pn_teleop_pr2_arms_node" pkg="perception_neuron_tf_broadcaster" type="pn_teleop_pr2_arms_node" output="screen">
    <!-- <node name="pn_teleop_pr2_single_arm_node" pkg="perception_neuron_tf_broadcaster" type="pn_teleop_pr2_single_arm_node" output="screen"> -->
      <rosparam command="load" file="$(find perception_neuron_tf_broadcaster)/config/pos_parameters.yaml" />
      <param name="left_teleop" type="bool" value="false" />
      <param name="right_teleop" type="bool" value="true" />
      <param name="gripper_teleop" type="bool" value="false" />

      <param name="vel_method" type="bool" value="true" />
      <param name="demo_test" type="bool" value="false" />
      <param name="move_group_name" type="std::string" value="left_arm" />

      <param name="shoulder_rot" type="bool" value="false" />
      <param name="elbow_pos" type="bool" value="true" />
      <param name="fore_rot" type="bool" value="false" />
      <param name="wrist_pos" type="bool" value="false" />
      <param name="wrist_rot" type="bool" value="false" />
      <param name="wrist_pose" type="bool" value="true" />
    </node>
</launch>
